yarn.lock

```diff
@@ -2366,6 +2366,18 @@
     jest-util "^29.3.1"
     slash "^3.0.0"
 
+"@jest/console@^29.5.0":
```

Comment:
Tons of unrelated changes in this file aren't good.
Please revert the `yarn.lock` to the one from `main` branch. Then execute `yarn` and the result should be only the changes related to `vue-imask`.